### PR TITLE
Adding cask Younity

### PR DESCRIPTION
**Pull request by tshrjn**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name. (Don't know how to squash commits)
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by adidalal** [comment]
Please explicity set version & shasum.
Use '#{version}` in the pkg stanza as well

**Comment by tshrjn** [comment]
Added version in package stanza. (Y)

File: Casks/younity.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'younity' do
+  version '1.12.6'
+  sha256 :no_check
+
+  url 'http://downloads.getyounity.com/files/younityInstaller.dmg'
+  name 'Younity'
+  homepage 'https://getyounity.com'
+  license :gratis # TODO: change license and remove this comment; ':unknown' is a machine-generated placeholder
+
+  pkg 'younityInstaller-#{version}.pkg'
```
Comment (reitermarkus) [inline Casks/younity.rb]: You need to use double quotes when interpolating `#{version}`.

Comment (tshrjn) [inline Casks/younity.rb]: Done !

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in dac42138652113127f3e94c5c98b8c0650a5c89a. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by tshrjn** [comment]
Hey Adidalal
Since doing the update I'm facing an issue updating brew.
I am not so sure but I think I am working in my clone repo of caskrooom I made(made for making a branch in order to send a pull request to add this software) instead of the native caskroom.
Here's the response of "brew update":

```
fatal: ambiguous argument 'refs/remotes/origin/master': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
remote: Repository not found.
fatal: repository 'https://github.com/homebrew/homebrew-cask/' not found
Error: Fetching /usr/local/Library/Taps/caskroom/homebrew-cask failed!
fatal: Cannot update paths and switch to branch 'master' at the same time.
Did you intend to checkout 'origin/master' which can not be resolved as commit?
fatal: Needed a single revision
invalid upstream origin/master
Error: Could not link caskroom/cask manpages to:
/usr/local/share/man/man1/brew-cask.1

Please delete these files and run `brew tap --repair`.
```

**Comment by adidalal** [comment]
@tshrjn Looks like this is a casualty of our merge into Homebrew.

Delete your fork of homebrew-cask via github
`cd $(brew --repo); git fetch; git reset --hard origin/master; brew update`
do https://github.com/caskroom/homebrew-cask/blob/master/doc/reporting_bugs/pre_bug_report.md

and then follow https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md one more time. That should fix any issues

**Pull request closed without merging**